cuda/python/python3.10/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ghcr.io/neuralmagic/base-cuda11.1:latest
+
+RUN set -Eeuxo \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+       wget python3-pip\
+       libreadline-gplv2-dev libncursesw5-dev \
+       libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://www.python.org/ftp/python/3.10.0/Python-3.10.0.tgz  \
```

Comment:
Maybe use `ADD` instead of wget?

Reply:
Actually don't use add as that would be a separate layer, just combing the statements should be fine